README.md

```diff
@@ -4,7 +4,7 @@
 
 # Welcome!
 
-This repository contains the source code of Cascadia Code.
+This repository contains the source code of the Cascadia Code.

```

Comment:
"Cascadia Code" refers to the family of fonts in this case. Adding "the" doesn't really make sense to me.